migaloo/assetlist.json

```diff
@@ -83,6 +83,41 @@
       "logo_URIs": {
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/migaloo/images/fable.svg"
       }
-    }
+    },
+      {
+        "description": "boneLUNA are autocompounding LUNA tokens",
+        "denom_units": [
+          {
+            "denom": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+            "exponent": 0
+          },
+          {
+            "denom": "bluna",
+            "exponent": 6
+          }
+        ],
+        "base": "ibc/40C29143BF4153B365089E40E437B7AA819672646C45BB0A5F1E10915A0B6708",
+	    "type_asset": "ics20",
+        "name": "boneLuna",
+        "display": "bluna",
+        "symbol": "bLUNA",
+        "traces": [
+          {
+            "type": "ibc",
+            "counterparty": {
+              "chain_name": "terra2",
+              "base_denom": "boneLuna",
```

Comment:
boneLuna is not the base denom. It would be cw20:terra17aj4ty4sz4yhgm08na8drc0v03v2jwr3waxcqrwhajj729zhl7zqnpc0ml